src/main/java/org/apache/commons/text/similarity/SorensenDicesSimilarity.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.text.similarity;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ *
+ * @since 1.7
+ */
+public class SorensenDicesSimilarity implements SimilarityScore<Double> {
+
+    /**
+     * @param left  the first CharSequence, must not be null
+     * @param right the second CharSequence, must not be null
+     * @return result similarity
+     * @throws IllegalArgumentException if either CharSequence input is {@code null}
+     */
+
+    @Override
+    public Double apply(final CharSequence left, final CharSequence right) {
```

Comment:
And we also need documentation here. Other similarities have some sort of example, showing how the API is used, and what's the likely output given a certain output. Take a look at the new JaroWinkler for instance.

Reply:
yup, will push another commit with documentation .

Reply:
Done